compiler/testData/codegen/box/delegatedProperty/delegateToAnotherMutable.kt

```diff
@@ -0,0 +1,13 @@
+// WITH_REFLECT
```

Comment:
Looks like `WITH_REFLECT` is not needed in this test.

Also, it fails on the old JS backend (`JsCodegenBoxTestGenerated$DelegatedProperty.testDelegateToAnotherMutable`), please add `IGNORE_BACKEND: JS`